src/main/java/com/mitchej123/hodgepodge/mixins/Mixins.java

```diff
@@ -29,8 +29,13 @@ public enum Mixins {
             Side.CLIENT,
             () -> Common.config.fixPotionEffectRender,
             TargetedMod.VANILLA),
-    FIX_IMMOBILE_FIREBALLS(
-            "minecraft.MixinEntityFireball", () -> Common.config.fixImmobileFireballs, TargetedMod.VANILLA),
+    LONGER_CHAT(
```

Comment:
Why is this removed?

Reply:
must be an error when I merge master into my chat_enhancements branch which was pretty old